webapp/platform/types/src/theme.ts

```diff
@@ -0,0 +1,35 @@
+export type ThemeKey = 'denim' | 'sapphire' | 'quartz' | 'indigo' | 'onyx';
+
+export type LegacyThemeType = 'Mattermost' | 'Organization' | 'Mattermost Dark' | 'Windows Dark';
+
+export type ThemeType = 'Denim' | 'Sapphire' | 'Quartz' | 'Indigo' | 'Onyx';
+
+export type Theme = {
+    [key: string]: string | undefined;
```

Comment:
If this doesn't cause any type issues, would you mind removing this? I don't 100% remember why we made it so that you could add arbitrary values to the Theme.

If it's because of something we add in `utils/constants` versions of the themes, let's remove that from this version and we can fix it when we go to use this type in Channels